conans/test/functional/configuration/test_profile_config.py

```diff
@@ -0,0 +1,110 @@
+import textwrap
+
+import pytest
+
+from conans.test.utils.tools import TestClient
+
+
+@pytest.fixture
+def client():
+    client = TestClient()
+    conanfile = textwrap.dedent("""
+        from conans import ConanFile
+        from conan.tools.microsoft import MSBuild
+
+        class Pkg(ConanFile):
+            settings = "os", "arch", "compiler", "build_type"
+            def build(self):
+                ms = MSBuild(self)
+                self.output.info(ms.command("Project.sln"))
+        """)
+    client.save({"conanfile.py": conanfile})
+    return client
+
+
+@pytest.mark.tool_visual_studio
+def test_msbuild_no_config(client):
+    profile = textwrap.dedent("""\
+        [settings]
+        os=Windows
+        arch=x86_64
+        compiler=Visual Studio
+        compiler.version=15
+        compiler.runtime=MD
+        build_type=Release
+        """)
+    client.save({"myprofile": profile})
+    client.run("create . pkg/0.1@ -pr=myprofile")
+    assert "/verbosity" not in client.out
+
+
+@pytest.mark.tool_visual_studio
+def test_msbuild_config(client):
+    profile = textwrap.dedent("""\
+        [settings]
+        os=Windows
+        arch=x86_64
+        compiler=Visual Studio
+        compiler.version=15
+        compiler.runtime=MD
+        build_type=Release
+        [conf]
+        tools.microsoft.MSBuild:verbosity=Minimal
```

Comment:
I wanted to discuss if it should be ``conan.tools.microsoft.MSBuild:verbosity=Minimal``, to fully match the ``import``. Also, we will need other names for configuration for upload, and other stuff, not in the ``conan.tools`` domain. I think this can wait a bit, until the first full proposal is there.

Reply:
Keep in mind that only the "built-in" classes will be python `imports` and that lots of custom generators, toolchains, and other packages will be brought in through `python_requires`.  So, whenever we think about addressing items, we should always be thinking about two cases (and thus always two different syntaxes):
"packaged resources"
"built-in resources"